### PR TITLE
Sunkast patch 2

### PR DESCRIPTION
**Pull request by sunkast**
Scroll Wayfarer version on  hover.

### CODE REVIEW

File: wayfarer-version-display.user.js [hunk 1]
```diff
@@ -44,18 +45,21 @@
...
+                boxdiv.appendChild(box);
             });
         }
         try {
             const json = JSON.parse(this.response);
             if (!json) return;
             if (json.version) {
-                box.textContent = `v${json.version.replace('release-wayfarer-web-', '')}`;
+                box.textContent = `${json.version.replace('release-wayfarer-web-', '')}`;
+                box.title = `${json.version.split("-")[3]+"-"+json.version.split("-")[4]}`
```
Comment (NvlblNm) [inline wayfarer-version-display.user.js:62]: fyi the `` `${stuff}` `` syntax is a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) so:
- line 61 doesn't really need that, it can just be `stuff` instead since the `v` is gone
- line 62 should ideally use it properly rather than mixing it with string concatenation. It could also just be `stuff` and use string concatenation, but IMO template literals are more readable. In this case, it would look like: `` `${json.version.split("-")[3]}-${json.version.split("-")[4]}` ``

That said, `box.textContent` and `box.title` are being set to exactly the same thing today, with brittleness in different areas. For `textContent`, the `replace` call will do nothing if Wayfarer releases a new version that doesn't have that exact string, while for `title`, if the number of dashes changes, the result will too.

If Niantic releases:
- `release-wayfarer-web-5.1.2` tomorrow, the `textContent` will be `5.1.2` while the `title` will be `5.1.2-undefined` - in this case, clearly the first approach is preferable
- `release-opr-web-5.1-abcdef12` tomorrow, the `textContent` will be that full string, while the `title` will be `5.1-abcdef12` - in this case, clearly the second approach produces better results, but only because the number of dashes remained the same
- `5.1-abcdef12` tomorrow, the `textContent` will be that full string, while the `title` will be `undefined-undefined` - in this case, clearly the first approach is preferable
- `release-wayfarer-web-some-junk-5.1-abcdef12` tomorrow, the `textContent` will be `some-junk-5.1-abcdef12`, while the `title` will be `some-junk` - in this case, clearly the first approach is preferable

In short, I would personally be inclined to consistently use `replace` instead of two different methods. You could either:
- create something like `const cleanVersion = ...` and assign it to both properties
- assign the second property from the first (i.e. `box.title = box.textContent`
